procurement_jit_assign_move/models/stock_move.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.multi
+    def action_confirm(self):
+        res = super(StockMove, self).action_confirm()
+        make_to_stock_moves = self.filtered(
+            lambda x:
+            x.procurement_id.rule_id.procure_method == 'make_to_stock'
```

Comment:
@jco-odoo for no good reason... I'd totally forgotten that field on the move. :blush: 

Thanks for your feedback.